.husky/pre-push

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn test
+yarn lint && yarn test
```

Comment:
I am now strongly against pre commit hooks as it ruins your life when rebasing.

Reply:
I'm ok with this